sheetstorm/settings/base.py

```diff
@@ -268,3 +268,5 @@
 ]
 
 CRISPY_TEMPLATE_PACK = 'bootstrap4'
+
+VALID_EMAIL_DOMAIN_LIST = ["codepoets.it"]
```

Comment:
Please set here `None` value and add comment what this settings is for. Real value of this setting please add to `development.py` and `production.py` settings. I think that this settings should be customizable and in base.py should be `None`.

Reply:
Maybe empty list instead of None? This way the want fail without setting it.

Reply:
Done

Reply:
Sure, empty list is better :)